quotas-and-limits_5cc850ba2c7d3a026fd42e7e.md

```diff
@@ -0,0 +1,81 @@
+Semaphore 2.0 enforces usage quotas and limits to protect customers and the
+platform from unforeseen spikes in usage. As your use of Semaphore expands over
+time, your quotas may increase accordingly.
+
+If you expect a notable upcoming increase in usage, you can proactively request
```

Comment:
To me this phrase "a notable upcoming release in usage" doesn't sound like what we usually say during customer calls. We need to address the primary reason for increasing default quotas, which is to support organizations whose needs are simply bigger than this.

Reply:
Updated this paragraph, and moved it under the machine quota section.

The new text says:

> If your organization needs are bigger than what is provided with the default
machine quotas, you can request an increase by sending a request to
<support@semaphoreci.com>.